File: C Programming/problem on numbericals/problem9.c
/*
9. Accept single digit number from user and print it into word. 
	
	Author:Milind Nalawade
	
	Date:24/06/2021
	
	language : C programming
	
*/
#include<stdio.h>

// Logic
void Number(int value)
{
	switch(value)
	{
		case 1:printf("one");
					break;
		case 2:printf("two");
					break;
		case 3:printf("three");
					break;
		case 4:printf("four");
					break;
		case 5:printf("five");
					break;
		case 6:printf("six");
					break;
		case 7:printf("seven");
					break;
		case 8:printf("eight");
					break;
		case 9:printf("nine");
					break;
	}
}
//Entry point function
int main()
{
	int i=0;
	printf("Enter your number : ");
	scanf("%d",&i);
	if(i<0)
	{
		i=i*-1;
	}
	Number(i);
	return 0;
}	

File: Python Programming/problems on numbericals/problem9.py
'''
9.Accept single digit number from user and print it into word. 
	
	Author:Milind Nalawade
	
	Date:24/06/2021
	
	language : Python 3.9.0
	
'''
#=====
#Logic
#=====
def Number(no):
	
	if no==1:
		print("one")
	elif no==2:
		print("two")
	elif no==3:
		print("three")
	elif no==4:
		print("four")
	elif no==5:
		print("five")
	elif no==6:
		print("six")
	elif no==7:
		print("seven")
	elif no==8:
		print("eight")
	elif no==9:
		print("nine")
	else:
		print("Number out of range")

#=============
#Entry point function
#=============		
def main():
	
	i=int(input("Enter your number : "))
	
	if(i<0):
		i=i*-1
	Number(i)

#=======
#starter
#=======	
if __name__=="__main__":
	main()
